Realm/RLMArray.h

```diff
@@ -264,7 +263,7 @@ NS_ASSUME_NONNULL_BEGIN
  Returns a sorted `RLMResults` from the array.
 
  @param property    The property name to sort by.
- @param ascending   The direction to sort by.
+ @param ascending   The direction to sort.
```

Comment:
I think it was previously correct and appropriate to end on the preposition. "To sort by."